src/Components/Card.js

```diff
@@ -0,0 +1,38 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+Card.propTypes = {
+  time: PropTypes.node,
+  meds: PropTypes.arrayOf(
+    PropTypes.shape({ id: PropTypes.node, name: PropTypes.string })
+  ),
+}
+
+export default function Card({ time, meds }) {
+  return (
+    <Wrapper>
+      <time dateTime={time}>{time} UHR</time>
+      <Meds>
+        {meds.map(({ id, medName }) => (
```

Comment:
I thought it would only work something like {id, name}, but again I've learned something new! cool.